src/components/KCatalog/KCatalog.cy.ts

```diff
@@ -55,7 +55,7 @@ const largeDataSet = [
  */
 
 describe('KCatalog', () => {
-  function getItems(count) {
+  function getItems(count: number) {
```

Comment:
I wish it would fix test instability :)

Reply:
I'm going to open a ticket -- we need to figure out why `KCatalog.cy.ts` fails

Reply:
If you want, assign to me, I might have some cycles to do a deep dive/refactor